pages/create.js

```diff
@@ -11,8 +11,24 @@ const StyledBackLink = styled(StyledLink)`
 export default function CreatePlacePage() {
   const router = useRouter();
 
-  function addPlace(place) {
-    console.log("Place added (but not really...)");
+  async function addPlace(place) {
+    try {
+      const response = await fetch("/api/places", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(place),
+      });
+      if (response.ok) {
+        router.push("/");
```

Comment:
do I really want to return to the home page? Is it a good UX? let us discuss this

Reply:
Yes, it was in task to do so :) Thanks for the hint.